### PR TITLE
Cleanup terran units file

### PR DESCRIPTION
**Pull request by speed2CZ**
Some clean up + fix for UEF factory build arms

### CODE REVIEW

File: lua/terranunits.lua [hunk 1]
```diff
@@ -200,19 +199,7 @@ TLandUnit = Class(LandUnit) {}
...
-    -- once it is.
-    StartBeingBuiltEffects = function(self, builder, layer)
-        self:SetMesh(self:GetBlueprint().Display.BuildMeshBlueprint, true)
-        if self:GetBlueprint().General.UpgradesFrom  ~= builder:GetUnitId() then
-            self:HideBone(0, true)
-            self.OnBeingBuiltEffectsBag:Add(self:ForkThread(CreateBuildCubeThread, builder, self.OnBeingBuiltEffectsBag))
-        end
-    end,
-}
+TMassCollectionUnit = Class(MassCollectionUnit) {}
```
Comment (aeoncleanse) [inline lua/terranunits.lua]: Please confirm you have checked that the UEF Mexes A: Definitely have animations for all stages of upgrade and B: Removing this definitely doesn't cause visual errors.

Comment (speed2CZ) [reply lua/terranunits.lua]: My quick check confirms that mexes do upgrade normally. The rest of testing is up to the test teeam

**Comment by shalkya** [comment]
visually seems fine although i got this error that i had on version without the fix too.
WARNING: Error running OnPaused script in Entity ueb0302 at 1b6d2e08: ...documents\github\fa\units\ueb0302\ueb0302_script.lua(59): attempt to call method `SetGoal' (a nil value)
         stack traceback:
         	...documents\github\fa\units\ueb0302\ueb0302_script.lua(59): in function `StopArmsMoving'
         	...s\%USER%\documents\github\fa\lua\terranunits.lua(63): in function <...s\%USER%\documents\github\fa\lua\terranunits.lua:61>
WARNING: Error running OnPaused script in Entity zeb9602 at 21501b08: ...documents\github\fa\units\zeb9602\zeb9602_script.lua(59): attempt to call method `SetGoal' (a nil value)
         stack traceback:
         	...documents\github\fa\units\zeb9602\zeb9602_script.lua(59): in function `StopArmsMoving'
         	...s\%USER%\documents\github\fa\lua\terranunits.lua(63): in function <...s\%USER%\documents\github\fa\lua\terranunits.lua:61>

**Comment by speed2CZ** [comment]
Those warnings are form the unit scripts, unrelated to this PR. Make a new issue for those please.

**Pull request merged**